tests/unit/Formatters/GeoCoordinateFormatterTest.php

```diff
@@ -401,6 +401,21 @@ public function decimalMinuteSecondNotationProvider() {
 				'0.1',
 				'-0° 6\', 0° 6\''
 			),
+			'Bug T150085 with 1 second precision' => array(
```

Comment:
If someone looks at this code and they dont know about phabricator they have no idea what this means. Even for people somewhat involved with the project this can easily get obscure. Imagine it has a bugzilla id; new colleagues would be confused.